Lab1/styles/style.css

```diff
@@ -0,0 +1,131 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    --bg-color-white: rgb(253, 223, 223);
+    --bg-color-green: lightgreen;
+    --bg-color-blue: skyblue;
+    --main-black-color: black;
+    --main-white-color: white;
+}
+
+.wrapper {
+    display: grid;
+    width: 100%;
+    min-height: 100vh;
+    grid-template-rows: minmax(20vh, auto) minmax(10vh, auto) minmax(50vh, auto) minmax(
+            20vh,
+            auto
+        );
+    grid-template-columns: 25% 50% 25%;
+}
+
+.wrapper > * {
+    border: 1px solid var(--main-black-color);
+}
+
+.wrapper div p,
+.wrapper div ol {
+    padding: 14px;
+    font-size: 14px;
+}
+
+.nav-link {
+    list-style: none;
+    height: 40px;
+    width: 200px;
+    padding: 5px;
+}
+
+.nav-link a {
```

Comment:
you may want to use scss

.nav-link {
....
  a { 
    .... 
    }
    ....
}